boards/weact/stm32g474_core/weact_stm32g474_core.yaml

```diff
@@ -0,0 +1,23 @@
+identifier: weact_stm32g474_core
+name: WeAct Studio STM32G474 Core Board
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 128
+flash: 512
+supported:
+  - nvs
+  - pwm
+  - i2c
+  - gpio
+  - usb device
+  - counter
+  - spi
+  - watchdog
+  - dma
+  - can
+  - rtc
```

Comment:
Some ips are missing like rng,dac, ...

Reply:
In fairness I think the inspiration was https://github.com/zephyrproject-rtos/zephyr/tree/main/boards%2Fst%2Fnucleo_g474re which has similar gaps :)

Reply:
ok, we'll take care to fix it.